components/centraldashboard/app/api.ts

```diff
@@ -3,24 +3,47 @@ import express from 'express';
 
 import {KubernetesService} from './k8s_service';
 
+interface PlatformStatics {
+    [key: string]: PlatformMeta;
+}
+interface PlatformMeta {
+    logo: string;
+    title: string;
+}
+
+/** API routes exposing information from Kubernetes */
+function _platformStatics(platform: string): {meta: PlatformMeta|undefined} {
```

Comment:
I was originally thinking this would live in the client-side code but this works well also. This would be one advantage to using TS in the front-end as well is that we should share type definitions.

I would suggest expanding the interface definition of PlatformInfo inside of the `k8s_service` module. Then, rather than use the _platformStatics function, I would just define an instance of the platform objects that we intend to support. That seems like a more reader-friendly way of defining the configuration since this is effectively acting like an enum.